apps/portal/src/app/views/home/view.njk

```diff
@@ -1,17 +1,39 @@
 {% extends "views/layouts/main.njk" %}
+{% from "govuk/components/tag/macro.njk" import govukTag %}
 
 {% block pageHeading %}{% endblock %}
 
 {% block pageContent %}
     <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
-            <div class="govuk-grid-row">
-                <div class="govuk-grid-column-full">
-                    <h2 class="govuk-heading-l">
-                        {{ pageTitle }}
-                    </h2>
-                </div>
-            </div>
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l govuk-!-margin-top-6">
+                Application reference number
+            </h1>
+
+            <h2 class="govuk-heading-m">
+                1. Your documents
+            </h2>
+
+            <ul class="govuk-task-list">
```

Comment:
please just use the govuk tasklist macro and pass the document category list in that way